odl/space/pspace.py

```diff
@@ -573,6 +575,39 @@ def element_type(self):
         return ProductSpaceVector
 
 
+def _broadcast_arithmetic(op, raises=False):
+    """Return self operator other with broadcasting
+
+    Parameters
+    ----------
+    op : string
+        Name of the operator, e.g. ``'__add__'``.
+    raises : bool
+        True if the method should raise on failure, otherwise returns
```

Comment:
This sentence is logically wrong. It should be "If true, then ...", while yours means "If ..., then true".
Suggestion:
```
For ``True``, the function raises on failure, otherwise it returns ``NotImplemented``.
```